test/e2e/features/widget-flows.feature

```diff
@@ -1,5 +1,3 @@
-# https://oktainc.atlassian.net/browse/OKTA-545127
-@skip(browserName=/internet.*explorer/)
```

Comment:
Restored this test, but it's flaky, so marked all `e2e-saucelabs*` Bacon tasks as `OPTIONAL`